src/slider/tp-slider.ts

```diff
@@ -445,6 +467,70 @@ export class TPSliderElement extends HTMLElement {
 		this.removeAttribute( 'resizing' );
 	}
 
+	/**
+	 * Set Default Settings.
+	 */
+	setDefaultSettings() {
```

Comment:
@imranhsayed Lets remove this

Reply:
Hi @junaidbhura Could you please elaborate on the reason it should be removed?
I have explained why we need this in this video.
https://www.loom.com/share/ab3e77a705c649f094fabefc0dbc9a04?sid=6b3406e8-4dd3-4e96-a7ef-c637d9c806e9

Reply:
Thanks @imranhsayed I mentioned it in my comment below:

> Lets us mandate that all cases be explicitly added in responsive, even the default ones - this will prevent us from dealing with weird edge cases in the future - and also reduce the burden on this library